models/release.go

```diff
@@ -94,6 +102,10 @@ func (r *Release) TarURL() string {
 
 // APIFormat convert a Release to api.Release
 func (r *Release) APIFormat() *api.Release {
+	apiAttachments := make([]*api.Attachment, len(r.Attachments))
```

Comment:
This will only work if `r.Attachments` has been populated before calling `APIFormat()`. This means we will need to update each endpoint that uses `Release.APIFormat()` accordingly.

Reply:
I am populating the list in loadAttributes on line 71->78. Is that ok?

Reply:
Oh, I missed that. Never mind then